### PR TITLE
Move `Core/Enums.gd` to `Global/Global_Enums.gd`

### LINKED ISSUE #54: Should `Core/Enums.gd` be moved to `Global/GLOBAL_VAR.gd` ?
## Is your amelioration related to a problem? Please describe.

There are enums in both files, and that's pretty confused.

## Describe the solution you'd like
Move one file into another. I would at least like everything to be in `Global` though.

### PR DESCRIPTION
**Pull request by Turtyo**
# Description

As the title says. Also change the lines referencing the previous `Core/Enums`

## Related issue(s)

Closes #54 

## List of changes

- Move `Core/Enums.gd` to `Global/Global_Enums.gd`
- Change class name from `Enums` to `GlobalEnums`
- Move enum from `GLOBAL_VAR.gd` to `Global_Enums.gd`

## Tests

Same as previously.

### CODE REVIEW

**Review by JonaLam — DISMISSED** [review]
I see no probelms here, seems pretty good

**Review by cheesefrycook — APPROVED** [review]
Looks good, I like the GlobalEnums naming

File: Global/Global_Enums.gd [hunk 1]
```diff
@@ -55,3 +55,20 @@ enum LightLevel
...
+
+enum ENTITY_STAT_DICT_TYPE {
+	OFFENSE,
+	DEFENSE
+}
+
+# used to select the offense or defense dictionary for the stats of an entity
+# ! don't forget to update _ENTITY_STAT_DICT_SELECTOR in EntityStats.gd if you modify this
+
+enum POSSIBLE_MODIFIER_NAMES {
```
Comment (Tysterman74) [inline Global/Global_Enums.gd:67]: Just to be in-line with the naming convention in this file I would rename to `EntityStatDictType` and `PossibleModifierNames`

**Review by Tysterman74 — APPROVED** [review]
One comment, but otherwise we love our renaming folks <3

**Review by JonaLam — APPROVED** [review]

**Review by cheesefrycook — APPROVED** [review]

**Pull request merged**